src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -77,3 +127,336 @@ export async function getUBAFeeConfig(
     }
   );
 }
+
+export async function updateUBAClient(
```

Comment:
Yep! I've reworked it so that the update now regenerates up to the `MAX_BUNDLE_LOOKBACK_LENGTH` that's currently stored as a constant. (We could always make this an ENV var or pass it as an optional param).